### PR TITLE
When yurtctl revert executes yurtappmanager to delete automatically, no crd resource error is reported.

### LINKED ISSUE #557: When yurtctl revert executes yurtappmanager to delete automatically, no crd resource error is reported.
**What happened**:
If yurtappmanager is not deployed and yurtctl revert is executed, a notfound error will be reported when yurtappmanager is deleted.
**What you expected to happen**:

A not found error occurred
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- OpenYurt version:
- Kubernetes version (use `kubectl version`):
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**others**

/kind bug

### PR DESCRIPTION
**Pull request by yanyhui**
/kind bug
#### What this PR does / why we need it:
Yurtappmanager does not use the method of deleting crd resources when crd is not deployed.
#### Which issue(s) this PR fixes:

Fixes #557 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/yurtctl/cmd/revert/revert.go [hunk 1]
```diff
@@ -350,7 +350,6 @@ func removeYurtAppManager(client *kubernetes.Clientset, yurtAppManagerClientSet
 		return fmt.Errorf("fail to delete the deployment/%s: %s",
 			constants.YurtAppManager, err)
 	}
-	klog.Info("deployment for yurt app manager is removed")
```
Comment (adamzhoul) [inline pkg/yurtctl/cmd/revert/revert.go:353]: why do you think, we should remove that klog.Info?
same below.

File: pkg/yurtctl/cmd/revert/revert.go [hunk 2]
```diff
@@ -446,7 +436,7 @@ func removeYurtAppManager(client *kubernetes.Clientset, yurtAppManagerClientSet
 		return fmt.Errorf("fail to delete the UnitedDeploymentCRD/%s: %s",
 			"UnitedDeployment", err)
 	}
-	klog.Info("UnitedDeploymentcrd for yurt app manager is removed")
```
Comment (adamzhoul) [inline pkg/yurtctl/cmd/revert/revert.go:449]: removeYurtAppManager方法  -> removeYurtAppManager ?

File: pkg/yurtctl/util/kubernetes/util.go [hunk 3]
```diff
@@ -415,6 +415,17 @@ func DeleteCRDResource(clientset *kubernetes.Clientset, yurtAppManagerClientSet
...
 		dri = yurtAppManagerClientSet.Resource(mapping.Resource)
 	}
+
+	var uns *unstructured.Unstructured
+	uns, err = dri.Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	if uns == nil {
+		klog.Info("不存在 " + name + " 这种crd资源，不需要删除")
```
Comment (adamzhoul) [inline pkg/yurtctl/util/kubernetes/util.go:425]: use English only?

**Review by adamzhoul — CHANGES_REQUESTED** [review]

File: pkg/yurtctl/util/kubernetes/util.go [hunk 4]
```diff
@@ -415,6 +415,17 @@ func DeleteCRDResource(clientset *kubernetes.Clientset, yurtAppManagerClientSet
 	} else {
 		dri = yurtAppManagerClientSet.Resource(mapping.Resource)
 	}
+
+	var uns *unstructured.Unstructured
+	uns, err = dri.Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		return err
```
Comment (rambohe-ch) [inline pkg/yurtctl/util/kubernetes/util.go:422]: I think if err is a NotFound error, we need to return nil not err.

**Comment by rambohe-ch** [comment]
@yanyhui please merge two commits into one commit.

**Pull request closed without merging**